app/views/layouts/decidim/_process_header.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="process-header">
```

Comment:
Why we add this file?

Reply:
I didn't know it was being overwritten by "decidim-regulations" and found that adding this file was an easy solution.
Can we wait until we know for sure what shall we do with that custom model? I was dedicating time to updating it, but I stopped because I don't if it's time wasted.

Reply:
Finally, I modified this [other file](https://github.com/gencat/participa/blob/465da11b239f3e0df14567f8f92161881a655404/decidim-regulations/app/views/layouts/decidim/_process_navigation.html.erb), as currently it is broken down into multiple partials. It achieves the same behaviour.

Reply:
:+1: